.golangci.yml

```diff
@@ -17,6 +18,38 @@ linters:
   enable-all: true
   disable:
     # All these break for one reason or another
+    - nolintlint
+    - gocognit
+    - stylecheck
+    - testpackage
+    - goerr113
+    - exhaustivestruct
+    - errorlint
+    - wrapcheck
+    - paralleltest
+    - wsl
+    - godox
+    - tparallel
+    - gomnd
+    - nlreturn
+    - noctx
+    - nestif
+    - thelper
+    - predeclared
+    - thelper
+    - ifshort
+    - staticcheck
+    - forbidigo
+    - govet
+    - exhaustive
+    - whitespace
+    - unparam
+    - gofmt
```

Comment:
The list looks way to big to me. `gofmt` is a standard linter. If that doesn't work, we should really fix the underlying issue.

Reply:
I enabled `gofmt` `goimports` and `govet`.

Reply:
Thanks a lot!